### ISSUE TITLE
Port Module Filter

### ISSUE THREAD

**Issue by jlfranklin** [backdrop-ops/contrib#468]
**Name of the module, theme, or layout you are working on**
Module Filter

**Link to the drupal.org module, theme, or layout**
https://www.drupal.org/project/module_filter

**(Optional) Link to the drupal.org issue where you notified the Drupal maintainer(s) of this port**

**Do you need assistance with this port?**
Yes

An initial port is complete at https://github.com/backdrop-contrib/module_filter

**Comment by jlfranklin** [comment]
The module uses jquery.ba-bbq.js, formerly in Drupal core, it was removed at some point in the development of Backdrop.  I've included it in the Module Filter module for the time being.  I'd like others opinions on where it should live.  I see three major options:

1. Put it back in core.
2. Leave it where it is.
3. Create a jQuery BBQ module or a jQuery Grimoire module with a collection of commonly used jQuery code.
